apps/mark-scan/frontend/src/pages/print_page.tsx

```diff
@@ -18,6 +19,22 @@ export function PrintPage(): JSX.Element {
   assert(typeof ballotStyleId === 'string', 'ballotStyleId is not defined');
   assert(typeof precinctId === 'string', 'precinctId is not defined');
 
+  const printerTimer = useRef(0);
```

Comment:
same question

Reply:
What are you expecting this should be? I don't quite understand.

Reply:
oh, this was just bad line selection, I just meant same question as before, apologies